package com.shopsphere.role;

import java.util.UUID;
import java.util.Optional;

import org.springframework.data.jpa.repository.JpaRepository;


public interface RoleRepository extends JpaRepository<Role, UUID> {

    Optional<Role> findByName(RoleName role);
}
